addons/website/static/src/snippets/s_faq_collapse/000.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
```

Comment:
Not sure to understand what you're doing here  🤔

`00X.xml` files into the `static` folder are supposed to contain QWeb JS templates, loaded in JS, eg https://github.com/odoo/odoo/blob/master/addons/website/static/src/snippets/s_countdown/000.xml

Reply:
As the template has totally changed, I thought I should keep a backup. I removed it :)